demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -242,25 +249,19 @@ def upload(
         # Implemented at the ContentItem/Pack level rather than here
         raise NotImplementedError()
 
+    def get_related_content(self) -> Dict[RelatedFileType, dict]:
+        return {}
+
     @staticmethod
     @lru_cache
     def from_path(
         path: Path,
-        git_status: Optional[GitStatuses] = None,
-        old_file_path: Optional[Path] = None,
         git_sha: Optional[str] = None,
+        raise_on_exception: Optional[bool] = False,
```

Comment:
```suggestion
        raise_on_exception: bool = False,
```

no reason it would be an `Optional`